src/docfx/SubCommands/PdfCommand.cs

```diff
@@ -63,11 +63,12 @@ public void Exec(SubCommandRunningContext context)
                 ExcludeTocs = _config.ExcludedTocs?.ToArray(),
                 KeepRawFiles = _config.KeepRawFiles,
                 LoadErrorHandling = _config.LoadErrorHandling,
+                FilePath = _config.Wkhtmltopdf?.FullFilePath,
                 AdditionalPdfCommandArgs = _config.Wkhtmltopdf?.AdditionalArguments,
                 TocTitle = _config.TocTitle,
                 OutlineOption = _config.OutlineOption,
                 CoverPageTitle = _config.CoverPageTitle,
-                NoInputStreamArgs = _config.NoInputStreamArgs,
+                NoInputStreamArgs = _config.NoInputStreamArgs
```

Comment:
Personally prefer dangling commas for https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8